src/config.coffee

```diff
@@ -32,6 +32,11 @@ module.exports = {
         type : 'boolean'
         default : true
         description : "When beautifying on save, use the entire file, even if there is selected text in the editor. Important: The `beautify on save` option for the specific language must be enabled for this to be applicable. This option is not `beautify on save`."
+      beautifyFileExtensions :
+        title: "Do not Beautify these extensions"
```

Comment:
Title should resemble the key. So `beautifyFileExtensions` would have a title like `Beautify File Extensions`. It is useful when capitalization of certain words will not be predicted programmatically.
Any more information should be stored within the `description`, as you have done below.